packages/edit-site/src/components/editor/index.js

```diff
@@ -152,7 +152,12 @@ function Editor( { initialSettings } ) {
 
 	const secondarySidebar = () => {
 		if ( isInserterOpen ) {
-			return <InserterSidebar />;
+			return (
+				<InserterSidebar
+					rootClientId={ isInserterOpen?.rootClientId }
```

Comment:
It seems weird that isInserterOpen is not a boolean. Can we rename the selectors? Are these stable?

Reply:
The setting (action) is exposed as experimental (`__experimentalSetIsInserterOpened`), but the selector was not. I created a separate experimental selector to get the insertion point and make sure that the `isInserterOpen` selector remains boolean.